aiosmtpd/tests/test_main.py

```diff
@@ -92,43 +92,59 @@ def setuid(mocker):
 # region ##### Helper Funcs ###########################################################
 
 
-def watch_for_tls(ready_flag, has_tls, req_tls):
-    has_tls.value = False
-    req_tls.value = False
+def watch_for_tls(ready_flag, retq: MP.Queue):
+    has_tls = False
+    req_tls = False
```

Comment:
PyPy had problems with `multiprocessing.Value`, but works totally well with `multiprocessing.Queue`, so we changed to the latter.